### PR TITLE
[marvell-arm64] Update HWSKU files to EZBv1.06

### PR DESCRIPTION
**Pull request by pavannaregundi**
#### Why I did it
Easy bringup (EZB) 1.06 adds following features:
- User config support for ACL enhancements.
- Range check and validation changes in yang model.

Change also adds ONIE_PLATFORM_EXTRA_CMDLINE_LINUX of "mem=4G" as a workaround due to restriction in addressing beyond 32-bit value for a arm64-marvell_rd98DX35xx-r0 platform.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305
- [X] 202311

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: device/marvell/arm64-marvell_rd98DX35xx-r0/installer.conf [hunk 1]
```diff
@@ -1 +1 @@
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="default_hugepagesz=32M hugepages=4"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="default_hugepagesz=32M hugepages=4 mem=4G"
```
Comment (lguohan) [inline device/marvell/arm64-marvell_rd98DX35xx-r0/installer.conf:1]: why do we need this mem=4G? what is the purpose for this?

Comment (pavannaregundi) [reply device/marvell/arm64-marvell_rd98DX35xx-r0/installer.conf:1]: HWSKU represented by this platform can be loaded on 4G or 8G RAM boards. This change is restricting the memory to 4G, as we have some open kernel issues due to some hardware component having only 32-bit addressing. Once we have solution to them, we plan to remove this restriction.

**Comment by pavannaregundi** [comment]
@lguohan Can we merge this PR?

**Review by Blueve — APPROVED** [review]

**Review by prgeor — APPROVED** [review]

**Pull request merged**